controller/transfer/util.py

```diff
@@ -83,6 +84,7 @@ def convert_to_record_dict(
             )
             raise Exception("Upload conversion error", "Upload ran into errors")
         # ensure useable columns dont break the import
+        df = df.replace("\u0000", " ", regex=True)
```

Comment:
Just as suggestion for the UI:
If we add an information extraction on the attribute, the space will be shown a one token and an be labeled, which maybe confusing because the other spaces are not labelable (if this word exists :D ). Maybe we can add some 'Warning' or 'Note' to inform the user where did the space come from.

Reply:
I will add a notification

Reply:
After further discussion we decided to not add additional handling since the replacement is handling enough